scripts/ci/setup-dev-sso.sh

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env bash
+
+user_help () {
+    echo "Deploy in cluster keycloak and configure registration service to use it."
+    echo "options:"
+    echo "-sn, --sso-ns  Builds and pushes the operator to quay"
+    echo "-h,  --help              To show this help text"
+    echo ""
+    exit 0
+}
+
+read_arguments() {
+    if [[ $# -lt 2 ]]
+    then
+        echo "There are missing parameters"
+        user_help
+    fi
+
+    while test $# -gt 0; do
+           case "$1" in
+                -h|--help)
+                    user_help
+                    ;;
+                -sn|--sso-ns)
+                    shift
+                    DEV_SSO_NS=$1
+                    shift
+                    ;;
+                *)
+                   echo "$1 is not a recognized flag!" >> /dev/stderr
+                   user_help
+                   exit -1
+                   ;;
+          esac
+    done
+}
+check_commands()
+{
+    for cmd in "$@"
+    do
+        check_command "$cmd"
+    done
+}
+
+check_command()
+{
+    command -v "$1" > /dev/null && return 0
+
+    printf "please install '%s' before running this script\n" "$1"
+    exit 1
+}
+
+read_arguments $@
+
+if [[ -n "${CI}" ]]; then
+    set -ex
+else
+    set -e
+fi
+
+
+check_commands yq oc base64 openssl
+
+# Install rhsso operator
+SUBSCRIPTION_NAME=${DEV_SSO_NS}
+printf "installing RH SSO operator\n"
+    INSTALL_RHSSO="apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${DEV_SSO_NS}
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: og-rhsso
+  namespace: ${DEV_SSO_NS}
+spec:
+  targetNamespaces:
+  - ${DEV_SSO_NS}
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ${SUBSCRIPTION_NAME}
+  namespace: ${DEV_SSO_NS}
+spec:
+  channel: stable
+  name: rhsso-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Automatic"
+    echo "objects to be created in order to install operator"
+    cat <<EOF | oc apply -f -
+${INSTALL_RHSSO}
+EOF
```

Comment:
It seems something is broken with indentation, also -if you like- we can avoid the `cat` here

```suggestion

echo "objects to be created in order to install operator"
oc apply -f - <<< "${INSTALL_RHSSO}"
```

Reply:
The indendention works fine for me, but your syntax should work as well. I'll apply your suggestion. Thanks!

Reply:
changed in https://github.com/codeready-toolchain/toolchain-cicd/pull/113/commits/07761694b9f83d5e089c2a366f10a487e7934216